fitbenchmarking/fitbenchmarking
# Added recalculation of performance profiles

## Description of work

Fixes #1217 .

This PR is to allow performance profiles to be recalculated when deselecting or deselecting a solver from the dropdown.
In the compare table one dropdown allows to control both graphs.

## Testing Instructions



1. Run fitbenchmarking
2. Try and deselect the one that looks like the best solver from performance profiles - The rest of the lines should get recalculated
3. Try and deselect all solvers and then re-adding them back - nothing should break


## Checklist



- [x] My PR fully fixes the issue linked. If new issues have been created, what are they? New issue created  #1246 .
